.github/workflows/ci.yaml

```diff
@@ -83,8 +83,6 @@ jobs:
         uses: sonatype-nexus-community/nancy-github-action@v1.0.2
       - run: npm install
         name: Install node deps
-      - name: Run prettier
```

Comment:
Is the formatting now checked somewhere else?

Reply:
Yep, https://github.com/ory/kratos/blob/master/.github/workflows/format.yml

Reply:
We can also rollback https://github.com/ory/kratos/commit/20fdfe8ddfe8f2922a8fa6c544319ebd8fa5d524 if the `format` check needs to be kept as a pre-req for tests

Reply:
👍

Reply:
Take a look now please